### PR TITLE
[ZH] Complete translation of 'FAQ`

### PR DESCRIPTION
**Pull request by tab-64**
Translated the unfinished part into Chinese.

完成了 'FAQ.md' 的翻译，将未完成的部分翻译为中文

## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)

### CODE REVIEW

**Comment by Walavouchey** [comment]
please refrain from updating the branch yourself. this only adds unnecessary commits and a maintainer will do so anyway before merging.

**Review by TicClick — APPROVED** [review]
merging (no native review for one week or longer)

**Pull request merged**
